src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -548,15 +548,15 @@ function HistogramPlotWithControls({
       truncationConfigIndependentAxisMax
     ) {
       setTruncatedIndependentAxisWarning(
-        'Data has been truncated (light gray area) by range selection'
+        ' Data has been truncated (light gray area) by range selection'
       );
     }
   }, [truncationConfigIndependentAxisMin, truncationConfigIndependentAxisMax]);
 
   useEffect(() => {
     if (truncationConfigDependentAxisMin || truncationConfigDependentAxisMax) {
       setTruncatedDependentAxisWarning(
-        'Data may have been truncated (light gray area) by range selection'
+        ' Data may have been truncated (light gray area) by range selection'
```

Comment:
It would be nice not to need the leading space.  Can the Notification component take care of this?

Reply:
@bobular thank you for your review and comments 👍 I will try to make this at Notification component level.